annotator-scanner/src/main/java/edu/ucr/cs/riple/scanner/AnnotatorScanner.java

```diff
@@ -151,6 +169,45 @@ public Description matchMemberSelect(MemberSelectTree tree, VisitorState state)
     return Description.NO_MATCH;
   }
 
+  @Override
+  public Description matchLambdaExpression(
+      LambdaExpressionTree lambdaExpressionTree, VisitorState visitorState) {
+    Config config = context.getConfig();
+    if (!config.callTrackerIsActive()) {
+      return Description.NO_MATCH;
+    }
+    // for e -> Foo.bar(e), assume that method "baz()" has been overridden. We need to serialize the
+    // region (leaf of path in visitor state) for "baz()".
+    serializeImpactedRegionForFunctionalInterface(config, lambdaExpressionTree, visitorState);
```

Comment:
Ah, ok, so the serialized region is the lambda / method reference itself?  Does this "overlap" with the region for the containing method, or is that a separate region?

Reply:
Also, is the reason you don't serialize the called method `bar` here (like you do below for a method reference) because that will get handled when visiting the body of the lambda?

Reply:
@msridhar The serialized region is the region for the containing method. And yes exactly, the called method gets handled when visiting the body of lambda.

Reply:
Ok I get it now.